expo/lib/types.ts

```diff
@@ -0,0 +1,118 @@
+export interface StreamingPlatformRemote {
+  playMusic: (musicUri: string) => Promise<void>;
+  pause: () => Promise<void>;
+  play: () => Promise<void>;
+  next: () => Promise<void>;
+  prev: () => Promise<void>;
+  setVolume: (volume: number) => Promise<void>;
+  seekTo: (position: number) => Promise<void>;
+}
+
+export type Artist = {
+  name: string;
+  id: string;
+};
+
+export type Music = {
+  title: string;
+  artwork: string;
+  /* This could be an array of Artist, but due to SoundCloud's API only returning a formatted
+   string, this has to be a string for now. */
+  artists: string;
```

Comment:
Then why not doing it like this :
```suggestion
  artists: string | Artist[];
```